Makefile

```diff
@@ -381,14 +381,18 @@ catalog-push: ## Push a catalog image.
 verify-unchanged: ## Verify there are no un-committed changes
 	./hack/verify-unchanged.sh
 
-.PHONY: container-build 
```

Comment:
Don't we want to keep `container-build` for non community builds ?

Reply:
Yes, why not :)